src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/util/BlazorStressApp.js

```diff
@@ -0,0 +1,17 @@
+import { BlazorApp } from "./BlazorApp.js";
+
+export class BlazorStressApp {
+  static instance;
+
+  constructor() {
+    if (BlazorStressApp.instance) {
+      return BlazorStressApp.instance;
```

Comment:
This is pretty weird. Could we change the other code that calls `new BlazorStressApp()` to use `BlazorStressApp.instance` instead? Otherwise when you're reading other code, you get the impression that you're getting separate instances, and wouldn't guess you were actually sharing a singleton.

Then this special constructor logic could be removed.

Reply:
Done